resources/views/projects/details.blade.php

```diff
@@ -0,0 +1,46 @@
+@extends('layouts.app')
+
+@section('content')
+
+<div class="padded content">
+	<form class="ui form">
```

Comment:
this shouldn't be a form. consider making this a div or removing it (unless you need it).

Reply:
Lo dejé porque se le quita el formato. Ya traté con div y sin nada pero se le quita el formato a los inputs para search project. Que me recomiendas?